build.gradle

```diff
@@ -35,7 +35,7 @@ sourceCompatibility = 1.7
 group = "com.google.api"
 version = "0.0.0-SNAPSHOT"
 // url to the snapshot repository of a private maven repo.
-def privateSnapshotRepo = "http://104.197.230.53:8081/nexus/content/repositories/snapshots/"
+def privateSnapshotRepo = "https://oss.sonatype.org/content/repositories/snapshots/"
```

Comment:
Question: the new value doesn't look like a private maven repo.
Consider renaming -- also wondering do we really need to have a variable like this.